from PyQt5 import QtCore, QtWidgets

class Ui_AddHouse(object):
    def setupUi(self, Form):
        Form.setObjectName("Form")
        Form.resize(390, 540)
        Form.setMaximumSize(QtCore.QSize(390, 540))
        self.verticalLayoutWidget = QtWidgets.QWidget(Form)
        self.verticalLayoutWidget.setGeometry(QtCore.QRect(60, 40, 261, 455))
        self.verticalLayoutWidget.setObjectName("verticalLayoutWidget")
        self.formLayout = QtWidgets.QVBoxLayout(self.verticalLayoutWidget)
        self.formLayout.setContentsMargins(0, 0, 0, 0)
        self.formLayout.setSpacing(0)
        self.formLayout.setObjectName("formLayout")
        self.cityLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.cityLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.cityLabel.setObjectName("cityLabel")
        self.formLayout.addWidget(self.cityLabel)
        self.cityTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.cityTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.cityTextEdit.setObjectName("cityTextEdit")
        self.formLayout.addWidget(self.cityTextEdit)
        self.addressLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.addressLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.addressLabel.setObjectName("addressLabel")
        self.formLayout.addWidget(self.addressLabel)
        self.addressTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.addressTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.addressTextEdit.setObjectName("addressTextEdit")
        self.formLayout.addWidget(self.addressTextEdit)
        self.sizeLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.sizeLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.sizeLabel.setObjectName("sizeLabel")
        self.formLayout.addWidget(self.sizeLabel)
        self.sizeTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.sizeTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.sizeTextEdit.setObjectName("sizeTextEdit")
        self.formLayout.addWidget(self.sizeTextEdit)
        self.priceLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.priceLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.priceLabel.setObjectName("priceLabel")
        self.formLayout.addWidget(self.priceLabel)
        self.priceTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.priceTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.priceTextEdit.setObjectName("priceTextEdit")
        self.formLayout.addWidget(self.priceTextEdit)
        self.rentPriceLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.rentPriceLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.rentPriceLabel.setObjectName("rentPriceLabel")
        self.formLayout.addWidget(self.rentPriceLabel)
        self.rentPriceTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.rentPriceTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.rentPriceTextEdit.setObjectName("rentPriceTextEdit")
        self.formLayout.addWidget(self.rentPriceTextEdit)
        self.bedroomCountLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.bedroomCountLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.bedroomCountLabel.setObjectName("bedroomCountLabel")
        self.formLayout.addWidget(self.bedroomCountLabel)
        self.bedroomCountTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.bedroomCountTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.bedroomCountTextEdit.setObjectName("bedroomCountTextEdit")
        self.formLayout.addWidget(self.bedroomCountTextEdit)
        self.furnishedLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.furnishedLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.furnishedLabel.setObjectName("furnishedLabel")
        self.formLayout.addWidget(self.furnishedLabel)
        self.furnishedTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.furnishedTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.furnishedTextEdit.setObjectName("furnishedTextEdit")
        self.formLayout.addWidget(self.furnishedTextEdit)
        self.otherLabel = QtWidgets.QLabel(self.verticalLayoutWidget)
        self.otherLabel.setMaximumSize(QtCore.QSize(16777215, 20))
        self.otherLabel.setObjectName("otherLabel")
        self.formLayout.addWidget(self.otherLabel)
        self.otherTextEdit = QtWidgets.QTextEdit(self.verticalLayoutWidget)
        self.otherTextEdit.setMaximumSize(QtCore.QSize(16777215, 30))
        self.otherTextEdit.setObjectName("otherTextEdit")
        self.formLayout.addWidget(self.otherTextEdit)
        self.addHouseButton = QtWidgets.QPushButton(self.verticalLayoutWidget)
        self.addHouseButton.setObjectName("addHouseButton")
        self.formLayout.addWidget(self.addHouseButton)

        self.retranslateUi(Form)
        QtCore.QMetaObject.connectSlotsByName(Form)

    def retranslateUi(self, Form):
        _translate = QtCore.QCoreApplication.translate
        Form.setWindowTitle(_translate("Form", "Form"))
        self.cityLabel.setText(_translate("Form", "City"))
        self.addressLabel.setText(_translate("Form", "Address"))
        self.sizeLabel.setText(_translate("Form", "Size"))
        self.priceLabel.setText(_translate("Form", "price"))
        self.rentPriceLabel.setText(_translate("Form", "Rent price"))
        self.rentPriceTextEdit.setPlaceholderText(_translate("Form", "(Type 0 if this house is not for rent)"))
        self.bedroomCountLabel.setText(_translate("Form", "Bedroom Count"))
        self.furnishedLabel.setText(_translate("Form", "Furnished"))
        self.furnishedTextEdit.setPlaceholderText(_translate("Form", "(True or False)"))
        self.otherLabel.setText(_translate("Form", "Other"))
        self.otherTextEdit.setPlaceholderText(_translate("Form", "(etc.)"))
        self.addHouseButton.setText(_translate("Form", "Add House"))